src/NzbDrone.Common/Instrumentation/Sentry/SentryTarget.cs

```diff
@@ -109,6 +111,35 @@ private bool IsSentryMessage(LogEventInfo logEvent)
 
             if (logEvent.Level >= LogLevel.Error && logEvent.Exception != null)
             {
+                // don't report uninformative SQLite exceptions
+                // busy/locked are benign https://forums.sonarr.tv/t/owin-sqlite-error-5-database-is-locked/5423/11
+                // The others will be user configuration problems and silt up Sentry
+                var sqlEx = logEvent.Exception as SQLiteException;
+                if (sqlEx != null && (sqlEx.ResultCode == SQLiteErrorCode.Busy ||
```

Comment:
Would we want to refactor this out into an array of non-reported errors and just test inclusion with that array? Seems like it would be a bit cleaner.